### PR TITLE
add error handling to previous 

### PR DESCRIPTION
**Pull request by rf20008**
Right now, the bot previous does not have an error handling command. Since nextcord is backwards compatible, I added code that should handle errors :)

### CODE REVIEW

**Comment by TAG-Epic** [comment]
errors are technically not backwards compatible yet.

**Comment by rf20008** [comment]
I just looked at the code and changed it so it should be better now :)

File: main.py [hunk 1]
```diff
@@ -3,14 +3,26 @@
...
 
+@bot.event
+async def on_command_error(ctx,error):
+    if isinstance(error, errors.MissingRequiredArgument):
+        await ctx.channel.send("You are missing a required argument.")
+        return
+    if isinstance(error, errors.CommandNotFound):
+        await ctx.channel.send("This command does not exist.")
+        return
+    await ctx.channel.send("Your command raised an exception: " + str(error))
```
Comment (ooliver1) [inline main.py]: if elif elif else seems to be a better idea but LGTM for basic eh

Comment (rf20008) [reply main.py]: I'm going to fix the conflicts and use if elif elif else :)

**Comment by rf20008** [comment]
i'm going to close this PR and re-open a new one with the new function (so i can fix the conflicts).

**Pull request closed without merging**